apps/studio/src/common/formUtils.ts

```diff
@@ -209,17 +209,25 @@ export const commonYupValidation = {
   password: Yup.string().required("Password is required"),
   publicationDate: Yup.date().max(new Date(), "Cannot be a future date"),
   releaseDate: (releaseDate: string | undefined) => {
```

Comment:
Maybe we should rename the `releaseDate` argument to something that actually references that it's the provided minReleaseDate from the backend endpoint. Right now it kind of looks like the field value, which it's not. Maybe `minReleaseDateString`?